Formula/j/jsontoolkit.rb

```diff
@@ -0,0 +1,50 @@
+class Jsontoolkit < Formula
+  desc "Swiss-army knife library for expressive JSON programming in modern C++"
+  homepage "https://jsontoolkit.sourcemeta.com/"
+  url "https://github.com/sourcemeta/jsontoolkit/archive/refs/tags/v2.0.0.tar.gz"
+  sha256 "00f82f02166beabec80522e2bbc7b839ee9b7ccb631411c42e6fab65186e80ba"
+  license "AGPL-3.0-only"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = %W[
+      -DCMAKE_COMPILE_WARNING_AS_ERROR=OFF
+      -DJSONTOOLKIT_CONTRIB=OFF
+      -DJSONTOOLKIT_TESTS=OFF
+      -DJSONTOOLKIT_DOCS=OFF
+      -DBUILD_SHARED_LIBS=ON
+    ]
+
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<~EOS
+      #include <sourcemeta/jsontoolkit/json.h>
+      #include <sstream>
+      #include <iostream>
+      int main() {
+        std::istringstream input{"true"};
+        const sourcemeta::jsontoolkit::JSON document =
+          sourcemeta::jsontoolkit::parse(input);
+        assert(document.is_boolean());
+        assert(document.to_boolean());
+        std::cout << "JSON Toolkit works!" << std::endl;
+        return 0;
+      }
+    EOS
+    system ENV.cxx, "test.cpp", "-std=c++17", "-I#{include}",
```

Comment:
JSON Toolkit only officially supports C++20. While in your test you are not using anything that wouldn't compile, maybe better to use C++20 here to avoid confusion?